psknode/bin/scripts/pskApiHubLauncher.js

```diff
@@ -41,4 +43,29 @@ function startServer() {
     });
 }
 
+let retryTimeout = 100;
+function retry() {
+    setTimeout(() => {
+        retryTimeout = (retryTimeout * 2) % (10 * 60 * 1000);
+        startServer();
+    }, retryTimeout);
+}
+
+// If the apihub port is already in use, retry starting the server
+// For other kind of exceptions exit with code 1
+process.on('uncaughtException', (e) => {
```

Comment:
File "port in use" exceptions and retry starting the server.
Exit with code 1 for any other type of exceptions